packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js

```diff
@@ -69,15 +77,67 @@ function SidebarNavigationScreenGlobalStylesContent() {
 	// rendering the iframe. Without this, the iframe previews will not render
 	// in mobile viewport sizes, where the editor canvas is hidden.
 	return (
-		<BlockEditorProvider
-			settings={ storedSettings }
-			onChange={ noop }
-			onInput={ noop }
-		>
-			<div className="edit-site-sidebar-navigation-screen-global-styles__content">
+		<>
+			<BlockEditorProvider
+				settings={ storedSettings }
+				onChange={ noop }
+				onInput={ noop }
+			>
 				<StyleVariationsContainer />
-			</div>
-		</BlockEditorProvider>
+			</BlockEditorProvider>
+		</>
+	);
+}
+
+function SidebarNavigationScreenGlobalStylesFooter( { onClickRevisions } ) {
+	const { revisions, isLoading } = useGlobalStylesRevisions();
+	const { revisionsCount } = useSelect( ( select ) => {
+		const { getEntityRecord, __experimentalGetCurrentGlobalStylesId } =
+			select( coreStore );
+
+		const globalStylesId = __experimentalGetCurrentGlobalStylesId();
+		const globalStyles = globalStylesId
+			? getEntityRecord( 'root', 'globalStyles', globalStylesId )
+			: undefined;
+
+		return {
+			revisionsCount:
+				globalStyles?._links?.[ 'version-history' ]?.[ 0 ]?.count ?? 0,
+		};
+	}, [] );
+
+	const hasRevisions = revisionsCount >= 2;
+	const { modified } = revisions?.[ 0 ] || {};
+
+	if ( ! hasRevisions || isLoading || ! modified ) {
+		return null;
+	}
+
+	return (
+		<VStack>
+			<SidebarNavigationItem
```

Comment:
Do you reckon this should have a focus state like the style variation buttons (blue border)?
<img width="335" alt="Screenshot 2023-06-01 at 3 29 08 pm" src="https://github.com/WordPress/gutenberg/assets/6458278/d418dbfa-c77f-4b26-bfa7-a94ebae97354">

Reply:
Good idea! I like the idea of the blue outline... it might be worth looking at it in a separate PR, though, as it'd apply to all `SidebarNavigationItem`s?

Reply:
> it might be worth looking at it in a separate PR, though, as it'd apply to all SidebarNavigationItems?

Good call